src/common/types.ts

```diff
@@ -113,3 +113,11 @@ export enum AsyncOperationStatus {
 export type NamedCredential = {
   Endpoint: string;
 };
+
+export type DeployComponent = {
```

Comment:
Could be do something like this so DeployComponent type is reusable?
```suggestion
export type DeployComponent = {
  sf_devops__Source_Component__c: string;
  sf_devops__Operation__c: string;
  sf_devops__File_Path__c: string;
};

export type FormattedDeployComponent = DeployComponent & {
  Type?: string;
  Name?: string;
};
```

Reply:
Is reusable DeployComponent, Type and Name are optional in this case.